code/missionui/missionshipchoice.cpp

```diff
@@ -1033,27 +1033,27 @@ void ship_select_blit_ship_info()
 	{
 		int sum = fl2i(sip->max_hull_strength + sip->max_shield_strength);
 		if(sum <= 600)
-			strcpy_s(str, "Light");
+			strcpy_s(str, XSTR("Light", 1796));
 		else if(sum <= 700)
-			strcpy_s(str, "Average");
+			strcpy_s(str, XSTR("Average", 1797));
```

Comment:
Isn't this the same as 1792?

Reply:
In English, yes. I don't know if other languages have nuance between "average quality" and "average strength", though, and XSTRs are cheap.